### ISSUE TITLE
Need to build new container image

### ISSUE THREAD

**Issue by marcodelapierre** [nextflow-io/rnaseq-nf#21]
Hi @pditommaso , we need to build a new version 1.3 and push it to quay.io.

Version 1.2 is broken due to the Dockerfile update with micromamba, which initially had no PATH update.

I have already bumped the version here: https://github.com/nextflow-io/rnaseq-nf/blob/master/docker/Makefile

So it is just a matter of running the Makefile; if you want to give me admin access to quay.io/nextflow I can do it (I am `marcodelapierre` there as well), or otherwise I will leave it to you

**Comment by toniher** [comment]
Yeah, I noticed that 1.2 is also missing `ps`, which leads to issues running it with Tower-

**Comment by pditommaso** [comment]
```
The push refers to repository [quay.io/nextflow/rnaseq-nf]
3fd35ef02eda: Preparing 
61b7068cf278: Preparing 
503e642d5ebf: Preparing 
c3957bc87975: Preparing 
5f70bf18a086: Preparing 
0eb428f50e73: Waiting 
65734c4ecb14: Waiting 
5ed208d0a888: Waiting 
9f170dfecb6d: Waiting 
16dea3230ded: Waiting 
cb4596cc1454: Waiting 
denied: System is currently read-only. Pulls will succeed but all write operations are currently suspended.
```

This! 👆

**Comment by marcodelapierre** [comment]
This looks like a Quay issue. 
Maybe some server maintenance going on on their end?

**Comment by pditommaso** [comment]
Indeed, they had some maintenance activity. Solved via afbdd46b
